test/message/vm_caught_custom_runtime_error.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+require('../common');
+const vm = require('vm');
+
+console.error('beginning');
+
+// Regression test for https://github.com/nodejs/node/issues/7397:
+// This should not print out anything to stderr due to the error being caught.
```

Comment:
This is the tiniest of nits, but maybe say vm.runInThisContext() should not print anything... As it currently stands, it seems like nothing should be printed at all, or the `catch` won't print anything either.